.gitignore

```diff
@@ -260,9 +260,9 @@ paket-files/
 .DS_Store
 
 # dev files
-RetrospectiveExtension.FrontEnd/vss-extension.dev.json
```

Comment:
When the folder contains a `.` gitignore does not honor the file name. that is why I created a separate .gitignore file inside `RetrospectiveExtension.FrontEnd`